src/plugins/plot/tickUtils.js

```diff
@@ -4,6 +4,18 @@ const e10 = Math.sqrt(50);
 const e5 = Math.sqrt(10);
 const e2 = Math.sqrt(2);
 
+// A complete list of time units and their duration in milliseconds
```

Comment:
is this UTC only? If so, should we make a note of that in the comments or variable naming?

Reply:
Good point. Will do.